submodules/eps/EPS.py

```diff
@@ -0,0 +1,187 @@
+import logging
+import time
+import smbus
+import yaml
+
+from smbus2 import SMBusWrapper
+from submodules import telemetry
+from core.log import Log
```

Comment:
The `Log` and `Error` classes have been moved to a separate package called `helpers`. Update your branch from `master` and refactor your imports